lib/linguist/languages.yml

```diff
@@ -4306,6 +4308,7 @@ Shell:
   - ksh
   - mksh
   - pdksh
+  - pwsh
```

Comment:
This should be removed. The `Shell` entry refers specifically to POSIX/Bourne-derived shells, rather than any generic shell.

Reply:
[Apparently](https://github.com/github/linguist/pull/4073#discussion_r176947264), the PowerShell interpreter can also interpret Bash scripts. I was able to confirm under Ubuntu with a simple:
```bash
#!/usr/bin/env pwsh
echo "ok"
pwd
cd /
pwd
```

Reply:
Not sure this is actually used as a shebang for Bash in-the-wild (on GitHub.com) though...

Reply:
If so, users should be identifying it using `#!/bin/sh` or a traditional interpreter.

There's already enough inconsistencies in the shell world...

Reply:
`pwd` `echo` `cd /`, `ls`, `curl` all work on Windows PowerShell since Vista/2006 (without any Unix-y stuff installed). These are official registered aliases to actual powershell cmdlets. If you try bash-only syntax, that will fail.